configure

```diff
@@ -69,6 +70,7 @@ Options:
   -no-runtime-lib      Do not compile nor install the runtime support library
   -no-standard-headers Do not install nor use the standard .h headers
   -clightgen           Also compile the clightgen tool
+  -ignore-coq-version  Do not check the version of Coq
```

Comment:
@maximedenes maybe a different wording could be used here? "Try to compile even with unsupported Coq versions."